README.md

```diff
@@ -44,8 +44,21 @@ $ brew install dnsmasq
 $ sudo brew services start dnsmasq
 ```
 
+### What services might you have available? Let's find out! ###
```

Comment:
I believe rather than available this is what services have been started at some point with `brew services`.